docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,89 @@
+# Release 479 (TBD 2025)
+
+## General
+
+* Update JDK to 25.0.1 ({issue}`27117`)
+* Support automatic TLS certificate generation in the `ANNOUNCE` node discovery mode. ({issue}`27030`)
+* Fix planning failure of certain queries involving empty VALUES. ({issue}`21506`)
```

Comment:
From the point of view of the user, VALUES cannot be empty.

Reply:
@kasiafi PTAL

Reply:
>From the point of view of the user, VALUES cannot be empty.

True. How about "Fix planning failure of certain queries where part of the plan is optimized to empty values."?

Reply:
applied to "Fix query failure when one of the branches of a `UNION` is known to produce zero rows during query planning" as blow suggested, FYI